### PR TITLE
Add Polymesh Ledger Integration

### PR DESCRIPTION
**Pull request by adamdossa**
Adds details needed for Polymesh ledger integration.

### CODE REVIEW

**Comment by adamdossa** [comment]
The three PRs (including this one) below are related:
https://github.com/polkadot-js/common/pull/834
https://github.com/polkadot-js/apps/pull/4397
https://github.com/polkadot-js/ui/pull/427

**Review by jacogr — COMMENTED** [review]
Thank you, will merge after a bump of `@polkadot/networks` & `@polkadot/ui` (Don't worry about the bump yourself, I'll add them as soon as both are merged and CI has published the updated packages)

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**
